dockstore-common/src/main/scala/io/dockstore/common/WdlBridge.scala

```diff
@@ -347,10 +366,42 @@ class WdlBridge {
   def readFile(filePath: String): String = Try(Files.readAllLines(Paths.get(filePath)).asScala.mkString(System.lineSeparator())).get
 }
 
+object AbsolutePathResolver {
+  /**
+    * Let's say /main.wdl imports /sub1/sub1.wdl, which in turn imports /sub1/sub2.wdl. sub1.wdl's import statement will be
+    * `import sub2.wdl`, i.e., the import will be relative to sub1.wdl.
+    *
+    * At the top level, secondaryFiles map will have these keys:
+    *
+    * /sub1/sub1.wdl
+    * /sub1/sub2.wdl
+    *
+    * Convert sub2.wdl into /sub1/sub2.wdl
+    *
+    * Note: This copied then Scalified from
```

Comment:
Thought: other option is to move the method from the webservice into common and then DRY by re-using it.

Reply:
Done in upcoming push